src/WatchlistCard.js

```diff
@@ -2,14 +2,15 @@ import React, { Component } from 'react';
 import './WatchlistCard.css';
 
 export default class WatchlistCard extends Component {
-  constructor () {
-    super();
+  constructor (props) {
+    super(props);
   }
   render() {
     return (
       <li className="watchlist-card">
-        
-        
+      <h3>{this.props.card.name}</h3>
+      <p>Season {this.props.card.season}: 
```

Comment:
Awesome!
Next step: let's add some conditional rendering to the watchlist so the episode has the name of the tv show on it!